tests/test_undo.rs

```diff
@@ -260,11 +260,11 @@ fn test_git_push_undo_colocated() {
     // same result in a seemingly different way?
     insta::assert_snapshot!(get_branch_output(&test_env, &repo_path), @r###"
     main (conflicted):
-      - 0cffb614 AA
-      + 0a3e99f0 CC
-      + 8c05de15 BB
-      @git (behind by 1 commits): 0a3e99f0 CC
-      @origin (behind by 1 commits): 8c05de15 BB
+      - qpvuntsm 0cffb614 (empty) AA
```

Comment:
TODO: I probably want to add (hidden) to the short commit template to make this look less confusing.